tests/common/include/tests/common/tools/intersectors/helix_cylinder_intersector.hpp

```diff
@@ -164,6 +162,9 @@ struct helix_cylinder_intersector
 
         return ret;
     }
+
+    // Tolerance for convergence
+    scalar_type tol{1e-3f};
```

Comment:
Can we maybe name it differently, so that it becomes clear what it does when you use it?
```suggestion
    /// Tolerance for convergence
    scalar_type convergence_tol{1e-3f};
```

Reply:
Fixed